### PR TITLE
[JBIDE-20344] Remove trailing slash in key for hostname

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

**Comment by jcantrill** [comment]
@fbricon Please review

File: plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/security/OpenShiftSecureStorageKey.java [hunk 1]
```diff
@@ -47,14 +47,19 @@ public String getKey() {
 				.toString();
 	}
 
-	private String stripScheme(String value) {
+	private String sanatizeHost(String value) {
```
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/security/OpenShiftSecureStorageKey.java]: sanitizeHost

**Comment by fbricon** [comment]
LGTM after fixing the typo

**Comment by jcantrill** [comment]
@fbricon closing as merged and pushed to master

**Pull request closed without merging**